rohmu/delta/common.py

```diff
@@ -254,6 +254,9 @@ def add_success(self, n: int = 1, *, info: str = "add_success") -> None:
             logger.debug("%s %r -> %r", info, n, self)
         assert not self.final
 
+    def progress_metrics(self) -> dict[str, int]:
```

Comment:
it would be better to have a `TypedDict` as the return type (with `total=True`) as it will avoid incorrect assumptions about which keys are present

Reply:
included TypedDict for progress_metrics